tbot_contrib/swupdate/__init__.py

```diff
@@ -0,0 +1,26 @@
+import tbot
+import pathlib
+from tbot.tc import shell
+from tbot.machine.linux import Path
+
+
+@tbot.testcase
+def swupdateweb_lab(path_to_swu_file: Path, ip: str) -> None:
+    with tbot.acquire_lab() as lh, tbot.acquire_local() as lo:
+        """copy the files from the local host to the lab host and the runs the swupdate_script.py on the lab host"""
+        p = Path(lo, pathlib.Path.cwd())
```

Comment:
`cwd()` is not a safe bet here because tbot could be run from anywhere.  Instead, reference `__file__` which should always point to this file.  Like this:

```suggestion
        p = Path(lo, __file__).parent
```